README.md

```diff
@@ -60,7 +60,6 @@ Hinweise zur manuellen Installation und Einrichtung finden sich im [Wiki](https:
 
 Die folgenden Fehler lassen sich nicht im Code von RSScrawler beheben, sondern nur auf Systemseite:
 
-* Python _Levenshtein_ wird ausschließlich in der Suche per Webinterface/API von der _fuzzywuzzy_ Bibliothek verwendet, die notfalls auf eine langsamere Alternative ausweicht. Die Warnung beim Start, dass das Modul fehlt, lässt sich optional per `pip install python-Levenshtein` vermeiden.
```

Comment:
this should no longer be a problem since rapidfuzz provides prebuild wheels https://pypi.org/project/rapidfuzz/#files so it is not required to compile it on windows